src/features/content-explorer/item-list/__tests__/ItemListIcon.test.js

```diff
@@ -33,33 +37,37 @@ describe('features/content-explorer/item-list/ItemListIcon', () => {
                 hasCollaborations: true,
                 isExternallyOwned: true,
             },
-        ].forEach(props => {
+            // externalFolder
+            {
+                type: 'folder',
+                hasCollaborations: false,
+                isExternallyOwned: true,
+            },
+        ].forEach(rowData => {
             test('should render correct folder icon', () => {
-                const wrapper = renderComponent(props);
-
-                expect(wrapper.find('FolderIcon').length).toBe(1);
-                expect(wrapper.prop('isCollab')).toEqual(props.hasCollaborations);
-                expect(wrapper.prop('isExternal')).toEqual(props.isExternallyOwned);
-                expect(wrapper.prop('title')).toBeTruthy();
+                const wrapper = renderComponent(rowData);
 
+                expect(wrapper.find('IconCell').length).toBe(1);
                 expect(wrapper).toMatchSnapshot();
             });
         });
 
         test('should render correct file icon', () => {
-            const extension = 'boxnote';
-            const wrapper = renderComponent({ type: 'file', extension });
+            const rowData = { type: 'file', extension: 'boxnote' };
+            const wrapper = renderComponent(rowData);
 
-            expect(wrapper.find('FileIcon').length).toBe(1);
-            expect(wrapper.prop('extension')).toEqual(extension);
-            expect(wrapper.prop('title')).toBeTruthy();
+            expect(wrapper.find('IconCell').length).toBe(1);
+            expect(wrapper.prop('rowData')).toEqual(expect.objectContaining(rowData));
+            expect(wrapper).toMatchSnapshot();
         });
 
         test('should render correct bookmark icon', () => {
-            const wrapper = renderComponent({ type: 'web_link' });
+            const rowData = { type: 'web_link' };
+            const wrapper = renderComponent(rowData);
 
-            expect(wrapper.find('BookmarkIcon').length).toBe(1);
-            expect(wrapper.prop('title')).toBeTruthy();
+            expect(wrapper.find('IconCell').length).toBe(1);
+            expect(wrapper.prop('rowData')).toEqual(expect.objectContaining(rowData));
+            expect(wrapper).toMatchSnapshot();
```

Comment:
I don't think these snapshots test anything different than directly checking the props?

Reply:
You're totally right. I added them to match the existing snapshot test, but I think we can safely remove all of them.